File: C++/First Steps in Coding - Basics of C++/Nested Loops/Exercise/Train the Trainers.cpp
#include <iostream>
#include <string>
#include <iomanip>
using namespace std;

int main() {
	int n, presentations = 0;
	double totalGrade = 0;
	cin >> n;
	while (true) {
		double gradesSum=0, avgPresentGrade;
		string presentationName;
		getline(cin >> ws, presentationName);
		if (presentationName != "Finish") {
			presentations += 1;
			for (int k = 1; k <= n; k++) {
				double givenGrade;
				cin >> givenGrade;
				gradesSum += givenGrade;
			}
			avgPresentGrade = gradesSum / n;
			totalGrade += avgPresentGrade;
			cout << fixed << setprecision(2) << presentationName << " - " << avgPresentGrade << "." << endl;
		}
		else {
			double totalAvgGrade = totalGrade / presentations;
			cout << fixed << setprecision(2) << "Student's final assessment is " << totalAvgGrade << ".";
			break;
		}
	}

}

File: Python/Basics/Nested Loops/Exercise/Train The Trainers.py
totalAvgScore = 0
presentations = 0
jurySize = int(input())

while True:
    presentationName = input()

    if presentationName != "Finish":
        presentations += 1
        score = 0
        for k in range(0, jurySize):
            grade = float(input())
            score += grade
            totalAvgScore += grade

        print(f"{presentationName} - {score / jurySize:.2f}.")

    else:
        break

print(f"Student's final assessment is {totalAvgScore / (jurySize * presentations):.2f}.")